CONTRIBUTING.zh-CN.md

```diff
@@ -51,14 +51,46 @@
 $ git checkout -b branch-name
 
 # 开发完成后跑下测试是否通过，必要时需要新增或修改测试用例
-$ npm test
+$ npm run test
```

Comment:
> This runs a package's "test" script, if one was provided.

[`npm test`](https://docs.npmjs.com/cli/test) will execute the 'test' script. `run` is useless.